src/app/app.component.ts

```diff
@@ -37,6 +37,8 @@ export class AppComponent implements OnInit {
         this.snake.move();
         if (!this.isGameOver()) {
           this.eatFood();
+          this.eatSnake();
+
```

Comment:
I'd like to keep the empty line as a separator between per-turn actions and the recursion.